src/client/user/actions/index.ts

```diff
@@ -617,6 +618,7 @@ const bulkUrlStarted = (
   dispatch<void>(getUrlsForUser())
   dispatch<ResetUserStateAction>(resetUserState())
   // TODO: placeholder before we add progress bar
+  // TODO: trigger qr code generation
```

Comment:
Suggest to put this logic on the server side and have it trigger the QR code generation directly